style.css

```diff
@@ -0,0 +1,83 @@
+ul {
+  list-style-type: none;
+}
+
+h1 {
+  background-color: pink;
+  text-align: center;
+  font-family: 'Pacifico', cursive;
+}
+
+h2 {
+  text-align: center;
+}
+
+section.team img {
+  border-radius: 50%;
+}
+
+/* a:hover {
+  background-color: hotpink;
+} */
+
+a[href="#"] {
+  color:black;
+  text-decoration: none;
+}
+
+section.sub {
+  text-align: center
+}
+
+blockquote {
+  font-style: italic;
+}
+
+blockquote::first-letter {
+  font-size: 2em;
```

Comment:
I liked that you used the first-letter to make the first letter of the quote larger, has very nice affect.